aten/src/THCUNN/THCHalfAutoNumerics.cuh

```diff
@@ -32,7 +32,7 @@ inline __host__ __device__ double fmaxType(double x, double y) {
 // arithmetic functions
 
 inline __host__ __device__ THHalf abs(THHalf a) {
-  return THCNumerics<THHalf>::abs(a);
+  return static_cast<THHalf>(std::abs(a));
 }
```

Comment:
I think you can remove the whole function, because definition of abs in TH should be removed.